[[toc]]

# News

## 6/28/2022

### Added

- Option to select the opacity of the spots in spatial page
- Moved download section in result page to an individual download page

### Updated

- The tutorials due to the function changed in the spatial result page
- Contact information

### Fixed

- Fixed error while loading deconvolution results
- Fixed issue while loading the spots image under the scatter plot
- The number of spatial samples and brain regions

## 6/22/2022

### Added

- Hint for calculating DEGs
- Replaced sankey plots with pie charts
- Links with GEO & Synapse database on the browse page
- Hint for enrichment database version using Enrichr
- Example button when searching genes and filter DEGs in the query page

### Updated

- The homepage plot will arrange the layout to default
- Updated user & data statistics
- FAQ, Contact, Tutorials
- Color palette when ploting cell clusters and homepage

### Fixed

- Fixed issue while loading gene expression profile in scatter plot
- Fixed issue while calculating overlapping DEGs from query page

## 5/27/2022

### Added

- New homepage
- New browse data page

## 5/8/2022

### Added

Spatial page:

- Deconvolution result using CARD

## 5/1/2022

### Added

Spatial page:

- spots colored by layers/ clusters
- Spatially variable genes
- DEGs and pathway

## 4/27/2022

### Fixed

- Violin plot cell types order
- Violin plot color scale
- Violin plot data display error

## 4/1/2022

Enrichment analysis:

- Added debunce when filtering DEGs to avoid too many Enrichr API calls
- Added loading indicator when filtering DEGs

## 3/24/2022

### Added

- Visualization: violin plots for selectedgene expression
- Visualization: automatically set color scale of cell types
- Visualization: volcano plot to visualize differentially expressed genes

## 3/16/2022

### Added

- Visualization: added frequency barplots at cell clustering section
- scroll to top button

## 3/11/2022

### Added

- Data query: added search function for finding overlapping DEGs from multiple comparisons

## 12/28/2021

### Added

Six new publication were added to scREAD (in-development)

- Single-nucleus transcriptome analysis reveals dysregulation of angiogenic endothelial cells and neuroprotective glia in Alzheimer’s disease
- Overexpressing low-density lipoprotein receptor reduces tau-associated neurodegeneration in relation to apoE-linked mechanisms
- Single cell RNA sequencing of human microglia uncovers a subset associated with Alzheimer’s disease
- Multimodal single-cell/nucleus RNA sequencing data analysis uncovers molecular networks between disease-associated microglia and astrocytes with implications for drug repurposing in Alzheimer's disease
- Diverse human astrocyte and microglial transcriptional responses to Alzheimer's pathology
- Prior activation state shapes the microglia response to anti-human TREM2 in a mouse model of Alzheimer's disease

## 5/26/2021

### Added

- New design to display enriched genes, with links to UniProt and GeneCards databases

### Changes

- In dataset information section, changed description of the number of cells
- Temporarily disabled submit section due to server updates

### Fixed

- Download link for raw data in fst format

## 5/18/2021

Our protocol paper: [Use of scREAD to Explore and Analyze Single-cell and Single-nucleus RNA-Seq data for Alzheimer’s Disease](https://doi.org/10.1016/j.xpro.2021.100513) has been published on STAR protocols!

## 11/13/2020

The scREAD paper is published online in [iScience](<https://www.cell.com/iscience/fulltext/S2589-0042(20)30966-4?rss=yes>)!